libdlib/CMakeLists.txt

```diff
@@ -10,24 +10,15 @@ find_package(catkin REQUIRED)
 message(STATUS "Project source dir: ${PROJECT_SOURCE_DIR}")
 add_custom_target(
     	build_dlib187 ALL
-    		COMMAND  cmake -E chdir ${PROJECT_SOURCE_DIR} make -f Makefile.tarball && touch ${CATKIN_DEVEL_PREFIX}/${CATKIN_GLOBAL_SHARE_DESTINATION}/${PROJECT_NAME}/INSTALLED 
+    		COMMAND  cmake -E chdir ${PROJECT_SOURCE_DIR} make -f Makefile.tarball
+		 COMMAND cmake -E copy ${PROJECT_SOURCE_DIR}/common/lib/libdlib.a  ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_LIB_DESTINATION}/
```

Comment:
In the last release of cob_extern the packages libntcan, libpcan and libphidgets used ${CATKIN_DEVEL_PREFIX} in the same way, does this mean it works like this?